docs/reference/config.md

```diff
@@ -167,34 +167,24 @@ The following settings are available:
 `aws.client.endpoint`
 : The AWS S3 API entry point e.g. `https://s3-us-west-1.amazonaws.com`. The endpoint must include the protocol prefix e.g. `https://`.
 
-`aws.client.maxConcurrency`
-: :::{versionadded} 25.06.0-edge
-  :::
-: The maximum number of concurrent S3 transfers used by the S3 transfer manager. By default, this setting is determined by `aws.client.targetThroughputInGbps`. Modifying this value can affect the amount of memory used for S3 transfers.
-
 `aws.client.maxConnections`
-: The maximum number of open HTTP connections used by the S3 transfer manager (default: `50`).
+: The maximum number of open HTTP connections used by the S3 client (default: `50`).
 
 `aws.client.maxDownloadHeapMemory`
 : The maximum size for the heap memory buffer used by concurrent downloads. It must be at least 10 times the `minimumPartSize` (default:`400 MB`).
 
 `aws.client.maxErrorRetry`
 : The maximum number of retry attempts for failed retryable requests (default: `-1`).
 
-`aws.client.maxNativeMemory`
-: :::{versionadded} 25.06.0-edge
-  :::
-: The maximum native memory used by the S3 transfer manager. By default, this setting is determined by `aws.client.targetThroughputInGbps`.
-
 `aws.client.minimumPartSize`
 : :::{versionadded} 25.06.0-edge
   :::
-: The minimum part size used by the S3 transfer manager for multi-part uploads (default: `8 MB`).
+: The minimum part size used for multi-part uploads to S3 (default: `8 MB`).
```

Comment:
@jorgee do these multi-part settings apply only to uploads or also to downloads? I keep getting confused about this, it seems like they are used for both

Reply:
But I see that this setting is being used for downloads as well, such as `AwsS3Config::checkDownloadBufferParams()` and `ExtendedS3TransferManager::setDownloadBufferProperties()` ?

Reply:
Ah its true. It is also used in downloads to setup the initial buffer size, that by default is 10 times the part size. So it is used in all transfers

Reply:
Because if the threshold is only 8 MB then that would mean that our benchmarks with 1 GB files were using multipart